src/Components/Politicians/PoliticianCard/styles.js

```diff
@@ -0,0 +1,17 @@
+export const container = {
+  flex: '0 1 20%',
+  margin: '1%',
+  display: 'flex',
+  flexWrap: 'wrap',
+  alignItems: 'center',
+  textDecoration: 'none',
+};
+
+export const flexItem = {
```

Comment:
I'm not too happy with the `flexItem` name here

Maybe a name documenting the purpose?